CMakeLists.txt

```diff
@@ -299,6 +316,10 @@ if (NOT "${TARGET_ARCH}" STREQUAL "${CMAKE_SYSTEM_PROCESSOR}")
     # TODO i#1684: We don't have full support for targeting arm in i386 yet.
     message(FATAL_ERROR "Targeting ARM on i386 is not yet supported: i#1684.")
   endif ()
+  if (RISCV64)
+    # TODO i#xxx: We don't have full support for targeting RISC-V in i386 yet.
```

Comment:
I think #1684 is generic for cross-arch decoder support so it can be referenced here.